apps/wpcom-block-editor/src/wpcom/features/tracking.js

```diff
@@ -417,4 +437,12 @@ if (
 			return null;
 		},
 	} );
+
+	registerDelegateEventSubscriber(
+		'wpcom-block-editor-template-part-detach-blocks',
+		'before',
```

Comment:
Noting it seems that for this case it doesn't really matter if this is 'before' or 'after',  although it seems good to have this flexibility in general! 🎉   Its great that we can add extra handlers when we want, and of course the big thing here is being able to track on capture as well!